package.json

```diff
@@ -82,7 +82,7 @@
     "@bbc/gel-foundations": "1.2.0",
     "@bbc/psammead-assets": "^0.1.9",
     "@bbc/psammead-brand": "^1.0.1",
-    "@bbc/psammead-caption": "^1.0.1",
+    "@bbc/psammead-caption": "^1.1.3",
```

Comment:
This PR also fixes this issue https://github.com/bbc/simorgh/issues/1690 since it pulls in `psammead-caption@1.1.3` and associated visual styling changes. 

Could you please add that to the PR and clarify that the styling changes will also be visible in your PR description test notes.

Otherwise, the `psammead-caption` changes can be pulled into a separate PR.